versioned_docs/version-0.10/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,30 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.12.0
+DATE TBD
+
+### Improvements
```

Comment:
how about we sort the improvements based on how big their impact is? e.g Supporting "workloads" is big while handling timeouts is not that impactful.